### PR TITLE
Update EDU kit registration articles [HC-1517]

### PR DESCRIPTION
**Pull request by Renat0Ribeir0**

### CODE REVIEW

**Review by gorillagripcore — APPROVED** [review]
Just a few suggestions, otherwise LGTM 🌷

File: content/Education and Kits/Registration/Register-an-Arduino-Education-Kit.md [hunk 1]
```diff
@@ -3,16 +3,18 @@ title: "Register an Arduino Education Kit"
 id: 4407393580818
 ---
 
-Learn how to activate your kit and access the online learning content for your Arduino Education kit.
+ Most Arduino Education kits don't require registration, and all online learning content can be found by visiting [courses.arduino.cc](https://www.arduino.cc/education/courses/). However, some kits need to be registered to have full access to their respective courses. See [which kits need to be registered](https://support.arduino.cc/hc/en-us/articles/4402999992850-Where-is-the-activation-code-for-my-kit#kits).
```
[the remark this answers is not in the document]
Comment (Renat0Ribeir0) [reply content/Education and Kits/Registration/Register-an-Arduino-Education-Kit.md]: Resolved in [0aaaffa](https://github.com/arduino/help-center-content/pull/391/commits/0aaaffa17ab3004617dc513b6172334b58829857)

File: content/Education and Kits/Registration/Find-the-activation-code-for-an-Arduino-Education-kit.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Find the activation code for an Arduino Education kit"
+id: 4402999992850
+---
+
+Most Arduino Education kits do not require activation codes, and all online learning content can be found by visiting [courses.arduino.cc](https://www.arduino.cc/education/courses/). However, [some kits](#kits) require registration with an activation code for full access to their courses.
```
Comment (gorillagripcore) [inline content/Education and Kits/Registration/Find-the-activation-code-for-an-Arduino-Education-kit.md:6]:
```suggestion
Most Arduino Education kits do not require activation codes, and all online learning content can be found by visiting [courses.arduino.cc](https://www.arduino.cc/education/courses/). However, [some kits](#kits) require registration with an activation code for full course access.
```

**Review by gorillagripcore — APPROVED** [review]
Just a small phrasing suggestion, otherwise LGTM 🌷

**Review by g-castronovo — APPROVED** [review]
(=^•́ܫ•̀^=)〈^✪ᆺ✪^〉

**Pull request merged**